pkg/kube/client/client_test.go

```diff
@@ -62,3 +64,21 @@ func TestAddingDataField_ModifiesExistingObject(t *testing.T) {
 	assert.Contains(t, cm.Data, "new-field")
 	assert.Equal(t, cm.Data["new-field"], "value")
 }
+
+func TestDeleteConfigMap(t *testing.T) {
+	cm := configmap.Builder().
+		SetName("config-map").
+		SetNamespace("default").
+		Build()
+
+	client := NewClient(NewMockedClient())
+	err := client.CreateConfigMap(cm)
+	assert.NoError(t, err)
+
+	err = client.DeleteConfigMap(types.NamespacedName{Name: "config-map", Namespace: "default"})
+	assert.NoError(t, err)
+
+	_, err = client.GetConfigMap(types.NamespacedName{Name: "config-map", Namespace: "default"})
+	assert.Error(t, err)
+	assert.Equal(t, err, notFoundError())
+}
```

Comment:
Do we need both assert here?

I feel if the first one would fail, it will fail anyway in the second? If you think that's clearer this way is totally fine for me anyway!

Reply:
yep makes sense, we can just leave the second one in and remove the first! 👍